src/Makefile

```diff
@@ -220,9 +220,7 @@ LIBGPAC_FILTERS_FFMPEG=filters/ff_common.o filters/ff_avf.o filters/ff_dec.o fil
 LIBGPAC_FILTERS_LIBCAPTION=filters/dec_cc.o
 LIBGPAC_FILTERS_MPEGHDEC=filters/dec_mpeghdec.o
 
-ifeq ($(CONFIG_EMSCRIPTEN),yes)
```

Comment:
Is that expected?

Reply:
I guess not, there's no use of WebCodecs outside the Web right?

Reply:
it is si that we generate the doc for all platforms